src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -10,6 +10,10 @@
     },
     "frameworks": {
         "net451": {
+            "imports": ["net46"],
```

Comment:
Does this make sense?  I run on net451, but I import net46?

/cc @ericstj @anurse